sampling/tsv_resampling.py

```diff
@@ -145,7 +145,7 @@ def run_sobol_sampling(self, n_datapoints, tmp_csv_path, csv_path):
                 'year_bin_of_last_hvac'
             ],
             [
-                'cook'
+                'cook', 'ground_thermal_conductivity'
```

Comment:
can this variable be placed with other location variables (climate_zone, region, state, county, etc.? same for the other sampling file.